### ISSUE TITLE
[bug] [Minor] Creeper visual smoothing algorithm doesn't smooth out Creeper rise inside emitters when filled with AC

### ISSUE THREAD

**Issue by grabz-dev** [KnuckleCracker/CW4-bug-tracker#81]
In Ruins Repurposed, there are two emitters that output 300 every 10s. I noticed that on their own, without any AC present, this is the tallest that they ever visually appear during an emission:
![image](https://user-images.githubusercontent.com/51026936/95697169-1b45d880-0c3e-11eb-9aba-a466d569cb23.png)

However, when the area inside the emitter is filled by AC, sometimes spikes like this appear instantaneously:
![Peek 2020-10-12 03-49](https://user-images.githubusercontent.com/51026936/95697142-06694500-0c3e-11eb-95b2-51a085997941.gif)

If this is intentional, for example so that the player can immediately see that AC is gone from under their feet, this can be closed.

**Comment by knucracker** [comment]
Yeah, the smoothing algorithm is a net win, but it does have some downsides. It cuts out the 'noise' that a lot of people complained about in the early days of development (it was a common complaint made in the comments on videos I posted). That's the 'win' part. The downsides are that the creeper you see is not the creeper that exists. Surprisingly, nobody has really cared or noticed. I've tweaked it so that the differences are really hard to notice... with one exception. That exception is where there is a flip between C and AC.  It seemed better to 'reset' the smoothing whenever there was a flip. Without it, things like conversion orbitals, big AC bombing raids, or 4rpl induced AC/C conversion events, didn't have the visual impact that was desired.  So the treatment was to reset smoothing when there was a sign flip. That produces this effect. The consequence is that it also can pull back the curtain on the true height of the Creeper (or AC).
